src/System.Private.Xml/src/System/Xml/Xsl/Xslt/InvokeGenerator.cs

```diff
@@ -11,13 +11,13 @@ namespace System.Xml.Xsl.Xslt
     using T = XmlQueryTypeFactory;
 
     /**
-    InvokeGenerator is one of the trikest peaces here.
+    InvokeGenerator is one of the trickiest peaces here.
     ARGS:
          QilFunction func      -- Functions which should be invoked. Arguments of this function (formalArgs) are Let nodes
-                                  anotated with names and default valies.
-                                  Problem 1 is that default values can contain references to previouse args of this function.
-                                  Problem 2 is that default values shouldn't contain fixup nodes.
-         ArrayList actualArgs  -- Array of QilNodes anotated with names. When name of formalArg match name actualArg last one
+                                  annotated with names and default values.
+                                  Problem 1 is that default values can contain references to previous args of this function.
+                                  Problem 2 is that default values shouldn't contain fix up nodes.
```

Comment:
*fix-up* or *fixup* as a noun (including attributive noun, where a noun is used as an adjective). The OED cites both closed and hyphenated, but hyphenated seems to be the most commonly used by a large margin.

Reply:
@JonHanna reverted fix up. Thanks! :+1:

Reply:
Maybe *fix-up* as the more common, and also the least likely to trigger a spellchecker for someone else later on.